examples/catalog/src/Template.html

```diff
@@ -0,0 +1,12 @@
+<html>
```

Comment:
Template.html --> can you make the name less generic? It's not descriptive, since everything can be thought of as a "template"